.ci/README.md

```diff
@@ -1,150 +1,47 @@
-
-## I can't merge my PR!
-There are a couple reasons this might happen - I'll cover the ones I know about right now.
-
-* There might be merge conflicts right now - you'll see a failure in the first step with "there are merge conflicts" in the error message.  If you see that error message in Concourse, but *not* in GitHub, simply try again - the GitHub API sometimes reports merge conflicts for a minute or so after a new commit is pushed, even if there aren't any.
-
-* To make sure that you end up with a clean history, the Magician rebases your PR onto master, and so if an old commit of yours introduces a merge conflict, *even if you fix that conflict in a subsequent commit*, the rebase process will fail.  You need to fix this problem by rebasing your branch onto master yourself, and force-pushing to Github.
-
-* If you see Concourse complaining about an SSL error of some kind, please just retry.  The Magician will auto-retry most transient failures, but this one seems to be a failure way down in some C code, so the Magician can't retry it easily.
-
-## Other: Consult your local Magician Expert
-Remind them to update this page.
+## Deploying the pipeline
+The code on the PR's branch is used to plan actions - no merge is performed.
+If you are making changes to the workflows, your changes will not trigger a workflow run, because of the risk of an untrusted contributor introducing malicious code in this way.  You will need to test locally by using the [cloud build local builder](https://cloud.google.com/cloud-build/docs/build-debug-locally).
+If you are making changes to the containers, your changes will not apply until they are merged in and built - this can take up to 15 minutes.  If you need to make a breaking change, you will need to pause the pipeline while the build happens.  If you are making changes to both the containers and the workflows and those changes need to be coordinated, you will need to pause the build while the containers build and enforce every open PR be rebased on top of our PR.  It is probably better to build in backwards-compatibility into your containers.  We recommend a 14 day window - 14 days after your change goes in, you can remove the backwards-compatibility.
+
+## Design choices & tradeoffs
+* The downstreams share some setup code in common - especially TPG and TPGB.  We violated the DRY principle by writing separate workflows for each repo.  In practice, this has substantially reduced the amount of code - the coordination layer above the two repos was larger than the code saved by combining them.  We also increase speed, since each Action runs separately.
+* The downstream push doesn't wait for checks on its PRs against downstreams.  This may inconvenience some existing workflows which rely on the downstream PR checks.  This ensures that merge conflicts never come into play, since the downstreams never have dangling PRs, but it requires some up-front work to get those checks into the differ.
```

Comment:
This feels like an actionable statement but I don't know what action should be taken. Are you referring to things like travis? If so, I'm not sure what "it requires some up-front work to get those checks into the differ" means. Can you rephrase it in terms of what the state of the world is right now?

Reply:
Clarified!